src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -44,9 +44,6 @@
 									)
 								  )">false</UnoForceHotReloadCodeGen>
 
-		<!-- Use Application.LoadComponent for HotReload scenarios -->
-		<UnoUseXamlReaderHotReload>true</UnoUseXamlReaderHotReload>
```

Comment:
Is this change related to the issue we are trying to solve?

Reply:
It is, indeed.